api/controllers/game/move.js

```diff
@@ -0,0 +1,18 @@
+// Request to make a move
+module.exports = async function (req, res) {
+  try {
+    const { saveGamestate, publishGameState, unpackGamestate } = sails.helpers.gamestate;
+    const { execute, validate } = sails.helpers.gamestate.moves[req.body.moveType];
```

Comment:
We need to verify that the requesting player is in the game. Instead of it sending `playedBy` in the req.body, this needs to be verified in the db. Essentially if 
```
req.session.usr === game.p0
```

Then `playedBy` should be p0, same goes for p1. But if the requesting user is neither, then we error with you are not a player in this game (there's an existing error for it in the legacy endpoints)